### PR TITLE
[13.0][MIG] scheduler_error_mailer: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0

Superseed: https://github.com/OCA/server-tools/pull/1772

Locked by:
- [x] `scheduler_error_mailer` https://github.com/OCA/server-tools/pull/2262
- [x] `scheduler_error_mailer` https://github.com/OCA/server-tools/pull/2264

Please @CarlosRoca13 and @pedrobaeza can you review it?

@Tecnativa TT27587

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check #2173

**Comment by StefanRijnhart** [comment]
Also, please check https://github.com/OCA/server-tools/pull/2174

**Comment by victoralmau** [comment]
Now, it's ready to review.

**Review by pedrobaeza — APPROVED** [review]
Code review

**Review by CarlosRoca13 — APPROVED** [review]

File: scheduler_error_mailer/models/ir_cron.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        my_cron = self.browse(job_id)
+
+        if my_cron.email_template_id:
+            # we put the job_exception in context to be able to print it inside
+            # the email template
+            context = {"job_exception": str(job_exception), "dbname": self._cr.dbname}
+
+            _logger.debug("Sending scheduler error email with context=%s", context)
+
+            self.env["mail.template"].browse(my_cron.email_template_id.id).with_context(
```
Comment (StefanRijnhart) [inline scheduler_error_mailer/models/ir_cron.py]: Not blocking, but I'm wondering if there is a reason that the id is rebrowsed.

**Review by StefanRijnhart — APPROVED** [review]
Great work on those 12.0 PRs!

**Pull request merged**